src/amberscript/parser.cc

```diff
@@ -986,8 +992,50 @@ Result Parser::ParseRun() {
     if (!pipeline->IsGraphics())
       return Result("RUN command requires graphics pipeline");
 
+    token = tokenizer_->NextToken();
+    if (!token->IsString() || token->AsString() != "AS")
+      return Result("missing AS for RUN command");
```

Comment:
The INDEXED case is yet to come.

Reply:
https://github.com/google/amber/issues/473